src/StaticOpenApiValidatorCache.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gertjuhh\SymfonyOpenapiValidator;
+
+use League\OpenAPIValidation\PSR7\ValidatorBuilder;
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory;
+
+final class StaticOpenApiValidatorCache
+{
+    private static PsrHttpFactory | null $psrHttpFactory = null;
+
+    /** @var array<string, ValidatorBuilder> */
+    private static array $validatorBuilder = [];
+
+    /**
+     * Install a cache pool for the OpenAPI Validator
+     *
+     * @see https://github.com/thephpleague/openapi-psr7-validator#caching-layer--psr-6-support
+     */
+    public static CacheItemPoolInterface|null $validatorCache = null;
```

Comment:
Could you add some information about the usage in the readme please?

Reply:
Done!

Reply:
Thanks!